dashboard/urls.py

```diff
@@ -24,23 +24,18 @@
 handler500 = dashboard_views.InternalServerError.as_view()
 
 urlpatterns = [
-    # our app urls
-    url(r'^$', dashboard_views.Home.as_view(), name='home'),
-    url(r'^api/docs/', include('rest_framework_swagger.urls')),
-    url(r'^api/v1/', include('api.urls')),
-    url(r'^push/', include('push.urls')),
-    url(r'^accounts/login/$', dashboard_views.Login.as_view(), name='login'),
-
     # 3rd-party app urls
     url(r'^accounts/', include('allauth.urls')),
+    url(r'^api/docs/', include('rest_framework_swagger.urls')),
 
     # django urls
     url(r'^admin/', admin.site.urls),
 ]
 
-# Make some error pages available under error/ for easier testing
-if settings.DEBUG:
-    urlpatterns += [
-        url(r'^error/404/$', dashboard_views.NotFound.as_view()),
-        url(r'^error/500/$', dashboard_views.InternalServerError.as_view()),
-    ]
+urlpatterns += i18n_patterns(
+    # our app urls
+    url(r'^$', dashboard_views.Home.as_view(), name='home'),
+    url(r'^api/v1/', include('api.urls')),
+    url(r'^push/', include('push.urls')),
+    url(r'^accounts/login/$', dashboard_views.Login.as_view(), name='login'),
+)
```

Comment:
Only wrapping **our** apps urls in `i18n_patterns` to preserve functionality of `allauth`, `rest_framework`, and `admin` urls.